### PR TITLE
Add Gaomon GM116HD

### PR DESCRIPTION
**Pull request by longnguyen2004**
Values extrapolated from GM156HD, and confirmed with real tablet.

### CODE REVIEW

**Review by jamesbt365 — CHANGES_REQUESTED** [review]
Please convert to the configuration format seen in master.

File: OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "Gaomon GM116HD",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 257.4,
+      "Height": 143.6,
```
Comment (jamesbt365) [inline OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json:6]: The height value should be 143.4 if the Max Y value below is correct with the calculation MaxY/lpmm (28680/200)

File: OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "Gaomon GM116HD",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 257.4,
+      "Height": 143.6,
+      "MaxX": 51480.0,
```
Comment (jamesbt365) [inline OpenTabletDriver.Configurations/Configurations/Gaomon/GM116HD.json:7]: Not sure if you manually guessed these, but these need to be checked in the tablet > tablet debugger if you haven't already.

**Comment by longnguyen2004** [comment]
Updated values (the previous one was taken from a reseller's website and it was slightly off, changed to [official values](https://gaomon.cn/Pen_Display/GM116HD_GM156HD.html#:~:text=11.6inch-,%E5%B7%A5%E4%BD%9C%E5%8C%BA%E5%9F%9F,256.32%20x%20144.18mm,-%E6%98%BE%E7%A4%BA%E5%88%86%E8%BE%A8%E7%8E%87))

**Review by jamesbt365 — COMMENTED** [review]
Good to go once a diagnostic is provided with the tablet plugged in and working and the tablet is added to the tablets.md in the root of the directory.

**Comment by longnguyen2004** [comment]
[diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/12436088/diagnostics.txt)
Updated the support table

**Review by jamesbt365 — APPROVED** [review]

**Comment by X9VoiD** [comment]
@InfinityGhost CI is stuck here again, I can't bypass the checks to merge this.

**Pull request merged**